coderd/coderd.go

```diff
@@ -76,6 +77,10 @@ type Options struct {
 	TailscaleEnable    bool
 	TailnetCoordinator *tailnet.Coordinator
 	DERPMap            *tailcfg.DERPMap
+
+	// Metrics related intervals.
+	MetricsCacheRefreshInterval time.Duration
+	AgentStatsReportInterval    time.Duration
```

Comment:
I think `AgentStatsReportInterval` could become `AgentStatsRefreshInterval` too, just for consistency.